README.md

```diff
@@ -108,6 +108,10 @@ Some tests need a redis deployed in `localhost:6379`. You can run it in Docker w
 ```bash
 docker run --rm --net=host -it redis
 ```
+on MacOS invoke Docker thus:
+```bash
+docker run --rm -p 6379:6379 -it redis
```

Comment:
We can document this as the only option. It also works on linux and it's a better option. `--net=host` is more useful when you want to have connectivity between containers for example :+1:

Reply:
The commit and PR messages don't match the changes you made